index.js

```diff
@@ -31,6 +31,17 @@ class DetailsMenuElement extends HTMLElement {
     return input instanceof HTMLInputElement ? input : null
   }
 
+  clearFocus() {
+    clearFocus(this)
```

Comment:
A follow up commit (in this PR or a new one) to inline `clearFocus` would be good for readability.